.gitmodules

```diff
@@ -4,3 +4,8 @@
 [submodule "backend/extern/afv-native"]
 	path = backend/extern/afv-native
 	url = https://github.com/pierr3/afv-native.git
+[submodule "backend/extern/libuiohook"]
```

Comment:
Please add the submodule init command to the CI and release, since it won't build otherwise